content/silver/concepts/layout.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Layout
+weight: 400
+---
+
+{{< toc >}}
+
+Most scanner generators (such as Flex) require one to specify a global set of ignored terminals.
+Silver (and Copper) do things differently, since sometimes we want embed one language in another that has different notions of comments/whitespace.
+
+## Production layout
+
+The ignored whitespace (the "layout") of a production is a set of ignored terminals inserted between every element on the production's right-hand side. So:
+
+```
+Decl ::= 'typedef' Type Identifiers ';'
+```
+
+will (in effect) be expanded to
+
+```
+Decl ::= 'typedef' Layout Type Layout Identifiers Layout ';'
+```
+
+Note that before and after the first and last signature element there is no `Layout` added: that's the responsibility of the parent production, whatever that may be.
```

Comment:
You mention on another page that the layout before and after the root is the same as the layout of the root.  You should mention that here as well.